data/src/main/kotlin/com/antosik/benchproject/data/popular/movies/entity/remote/MovieResponseRemote.kt

```diff
@@ -1,19 +1,5 @@
 package com.antosik.benchproject.data.popular.movies.entity.remote
 
-import com.antosik.benchproject.data.movies.common.constants.MoviesConstants
-import com.antosik.benchproject.data.popular.movies.entity.database.MovieEntity
-
 internal data class MovieResponseRemote(
```

Comment:
do we need wrapping List<MovieRemote> into MovieResponseRemote ? i assume that this is how it looks on api site right ? if Yes, please ignore this comment :D

Reply:
### Response from API
<p float="left" align="center">
  <img src="https://user-images.githubusercontent.com/57491794/173552963-6cdb4f0f-17b7-457c-a9a0-e84f338f0265.png" width="500" title="TMDB get-popular-movies">
</p>

Reply:
With @BorysChajdas we had a similar discussion 😄 https://github.com/Nataniel-Antosik/Bench_Project/pull/10#discussion_r896474130